packages/jumpstarter-cli-admin/jumpstarter_cli_admin/install.py

```
@@ -1,25 +1,226 @@
+        f'⚠️  WARNING: This will permanently delete the "{cluster_name}" {cluster_type} cluster and ALL its data. Continue?'  # noqa: E501
+    ):
+        click.echo("Cluster deletion cancelled.")
+        return
+
+    if cluster_type == "kind":
+        await _delete_kind_cluster(kind or "kind", cluster_name)
+    elif cluster_type == "minikube":
+        await _delete_minikube_cluster(minikube or "minikube", cluster_name)
+
+
+async def _install_jumpstarter_helm_chart(
```

Pylint (3.3.7)

[refactor] Too many arguments (12/5)

(R0913)